Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM node:16-alpine
-
+FROM --platform=$BUILDPLATFORM node:16-alpine as builder
```

Comment:
This will build everything on the platform docker is running. i had issues running the same steps for arm on amd